.github/workflows/ci.yml

```diff
@@ -152,6 +152,9 @@ jobs:
         ember-try-scenario:
         - ember-lts-3.28
         - ember-lts-4.4
+        - ember-lts-4.8
+        - ember-lts-4.12
+        - ember-5-0
```

Comment:
Looks like this needs to be `ember-5.0`. Once that's fixed though, it looks like it should pass based on `ember-default` results

Reply:
@maxfierke ups my fail, in ember-try i have named `ember-5.0`

Reply:
can you start the workflow again?